### PR TITLE
Get CI green

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
## Summary

Just trying to get CI consistently green before proposing other changes.

## Details

There were two errors happening randomly but very often:

* A [timeout error](https://travis-ci.org/cucumber/cucumber-rails/jobs/461252725) error. Increasing selenium's timeout seems to fix it.

* [A problem](https://travis-ci.org/cucumber/cucumber-rails/jobs/459973208) with `geckodriver`. This seems like a problem with the `geckodriver-helper` gem, probably something equivalent to https://github.com/flavorjones/chromedriver-helper/issues/57. Since the gem was doing very little (download geckodriver and putting it in PATH), I decided to remove it.

## Motivation and Context

Get CI green to unblock other PRs.

## How Has This Been Tested?

My fork on TravisCI.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## Checklist:

- [ ] I've added tests for my code.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
Yeah, the only downside is that contributors now need to manually put `geckodriver` on their path. I added the install script to the contributing instructions so it's not too painful.

There's a separate alternative which does not have that issue: using the [webdrivers](https://github.com/titusfortner/webdrivers/) gem. I think this gem should not suffer from the same issues present before this PR because it does not put any binary on the user's path (and thus doesn't mess with rubygems binstubs) but instead downloads it and then configures selenium to use it.

I still think this solution is simple enough. Contributors similarly require `firefox` installed on their system for specs to work, and that's not solved by any gem. I think it's fine that external non ruby dependencies are setup manually.

File: features/step_definitions/cucumber_rails_steps.rb [hunk 1]
```diff
@@ -98,9 +97,12 @@ def fixture(path)
 Given /^I force selenium to run Firefox in headless mode$/ do
   selenium_config = %{
     Capybara.register_driver :selenium do |app|
+      http_client = Selenium::WebDriver::Remote::Http::Default.new
```
Comment (luke-hill) [inline features/step_definitions/cucumber_rails_steps.rb]: This isn't directly needed. But to hazard a guess is this extra timeout something that causes intermittent failures>?

Comment (deivid-rodriguez) [reply features/step_definitions/cucumber_rails_steps.rb]: Read the PR description! :)

**Comment by deivid-rodriguez** [comment]
I was about to add the `webdrivers` gem, but it also feels nice about this solution to consistently use a specific version of `geckodriver` instead of using always the latest. May I leave it as it is, and maybe iterate in the future if we end up considering the `webdrivers` gem a better approach?

**Comment by xtrasimplicity** [comment]
Let's go for it. :)

**Comment by aslakhellesoy** [comment]
Hi @deivid-rodriguez,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**